src/views/Correction.vue

```diff
@@ -877,6 +895,24 @@ section p {
   color: $gray7;
 }
 
+.correction-header {
+  font-size: 18px;
+  font-weight: bold;
+  color: $gray9;
+}
+
+
+.detail-comment-input {
+  :deep(placeholder) {
```

Comment:
`placeholder` is not a valid type selector (eg, div, p, li, etc). If it's a class then it need a `.`, or if it's an id then it needs a `#`.

Reply:
Ditto on line 933 below.